### ISSUE TITLE
pretrain model between my train model

### ISSUE THREAD

**Issue by Followmeczx** [IDEA-Research/OSX#117]
How is your OSX_l.pth pre-trained model set up for training？
I follow your setting1 and the training result so bad that the evaluation metrics such as MPVPE (All) is 20 different.
my train model:
![image](https://github.com/IDEA-Research/OSX/assets/65723649/b1785fca-ae33-4cc7-b3be-803e7d65d7cb)
OSX_l.pth:
![image](https://github.com/IDEA-Research/OSX/assets/65723649/38178ced-1325-4dab-91ed-4d19d49fd622)

**Comment by linjing7** [comment]
Hi, how many GPUs did you use? Have you loaded the model weights pretrained on COCO keypoints dataset? Could you please show me your training log?

**Comment by Followmeczx** [comment]
![image](https://github.com/IDEA-Research/OSX/assets/65723649/e0548b5a-f4c7-439e-8252-39cdad668dc0)
I trained on a single GPU, NVIDIA A100, for about a week.

**Comment by linjing7** [comment]
Hi, your batch size is 16 while our batch size is 64(16x4). Could you please try a larger batch size?

**Comment by Followmeczx** [comment]
You mean don't follow setting1. Instead, change batchsize to 64. Will this help the result much

**Comment by linjing7** [comment]
No, I means strictly following setting1. We use 4 GPU in setting 1 while you use 1. We set the batch size to 16 **per GPU**.

**Comment by Followmeczx** [comment]
Ok, thank you very much. I have one more question, will using wo_decoder not affect the result too much? I am not quite clear about the role of decoder for OSX. What does wo_decoder mean? Why can the human body be reconstructed without decoder

**Comment by linjing7** [comment]
It will regress the body, hand, and face param based on the encoder feature. The hand and face accuracy will be affected while the body accuracy will not be degraded obviously.

**Comment by Followmeczx** [comment]
Thank you very much! After change batch size ,The results are close to those of your pre-trained model。
